eth2prysm/values.yaml

```diff
@@ -2,21 +2,45 @@
 # This is a YAML-formatted file.
 
 namespace: prysm
-imageVersion: v1.3.3
 
 ethereumTestnet: pyrmont
 
+nfsEnabled: true
```

Comment:
note: comment available persistent storage options and take the string value directly

Reply:
Done.